.env.example

```diff
@@ -0,0 +1,2 @@
+db_host=mongodb://localhost:27017/letstalk
```

Comment:
```suggestion
db_host=mongodb://localhost:27017
```

Reply:
Updated above comment :)

Reply:
That works a "config.json" file, then in the main.go file it reads the son file and decodes it

Reply:
If we do define config variable in main.go we wont be able to use config variables in controller or model package as that could lead to an **import cycle** error.
Can we define the config in values package? say,

```
var Config = struct{
 VarA Type `json:"varA"`
 VarB Type `json:"varB"`
}{}
```

then have an [init](https://golang.org/doc/effective_go.html#init) function in values package that unmarshals the variable and panics if there's an error.

```
func init(){
   ...read config.json file and load as bytes

   if err:=json.Unmarshall(bytes,&Config);err!=nil{
     Log.Fatalln(discriptive error here)
   }
}
```

This would make adding more configurations easy and accessible to all core packages.